solutions/account-infrastructure-base/variables.tf

```diff
@@ -33,8 +33,13 @@ variable "prefix" {
   # this value was determined based on the lowest prefix restriction located here:
   # https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/blob/main/patterns/roks/variables.tf#L11
   validation {
-    condition     = (var.prefix == null || var.prefix == "") ? true : length(var.prefix) <= 13
-    error_message = "prefix` length must be 13 characters or less or null or an empty string (\"\")."
+    condition = (var.prefix == null ? true :
+      alltrue([
+        can(regex("^([a-z]|[a-z][-a-z0-9]{0,14}[a-z0-9])$|^$", var.prefix)),
```

Comment:
Need to add the {0,N} constraint on the first `all lowercase` check, also can you update the comment above this to show where the 16 is coming from? Currently we're restricting to 13 characters long so we should document here where the restriction change is coming from.